### PR TITLE
Des 90 functionality fixes

### PR DESCRIPTION
**Pull request by abder**

### CODE REVIEW

File: packages/assets/css/src/scss/_components/_checkbox/_elements/_checkbox-box.scss [hunk 1]
```diff
@@ -16,7 +16,8 @@
 			@include element(box) {
 				width: $checkbox-size-md;
 				height: $checkbox-size-md;
-				display: flex;
+				display: flex !important;
```
Comment (emgk) [inline packages/assets/css/src/scss/_components/_checkbox/_elements/_checkbox-box.scss:19]: @abder Is '!important' required can we try not to use it?

File: packages/assets/css/src/scss/_components/_selector/_states/_state-hover.scss [hunk 2]
```diff
@@ -12,9 +12,17 @@
...
-				0 $shadow-offset-default $shadow-offset-lg 0 $color-shadow-default;
+			@include modifier(hover) {
+				box-shadow: 0 $shadow-offset-xs $shadow-offset-sm 0 $color-shadow-dark,
+					0 $shadow-offset-2xs $shadow-offset-xl 0 $color-shadow-light,
+					0 $shadow-offset-default $shadow-offset-lg 0 $color-shadow-default;
+			}
+
+			@include element(option-delete-btn) {
+				&:hover {
+					background: #ab403e !important;
```
Comment (creador-dev) [inline packages/assets/css/src/scss/_components/_selector/_states/_state-hover.scss:23]: @abder Important and no color variable was used?

File: packages/assets/css/src/scss/_components/_selector/_elements/_selector-option.scss [hunk 3]
```diff
@@ -8,152 +8,148 @@
...
+
+				&::after {
+					border-left: unset;
+					border-right: $selector-border-width solid transparent;
+					border-top: $selector-border-width solid $color-primary-50;
+					transform: rotate(360deg);
+					right: unset;
+					left: -#{$spacing-sm};
+					border-top-right-radius: unset;
+					border-top-left-radius: 4px;
```
Comment (creador-dev) [inline packages/assets/css/src/scss/_components/_selector/_elements/_selector-option.scss:138]: @abder use $border-width-xl wherever we have 4px border. I must have missed this earlier.

File: packages/assets/css/src/scss/_components/_selector/_elements/_selector-option.scss [hunk 4]
```diff
@@ -8,152 +8,148 @@
...
+					position: absolute;
+					left: auto;
+					top: 0;
+					right: 0;
+					width: 0;
+					height: 0;
+					border-left: $selector-border-width solid transparent;
+					border-top: $selector-border-width solid $color-primary-50;
+					clear: both;
+					border-top-right-radius: 4px;
```
Comment (creador-dev) [inline packages/assets/css/src/scss/_components/_selector/_elements/_selector-option.scss:49]: @abder same

File: packages/assets/css/src/scss/_components/_selector/_elements/_selector-delete.scss [hunk 5]
```diff
@@ -8,33 +8,35 @@
...
-            @include element(option-delete-btn) {
-                background-color: $color-extended-error-50;
-                color: $color-extended-neutral-100;
-            }
-        }
-    }
+			@include element(option-delete-btn) {
+				background-color: $color-extended-error-50;
+				color: $color-extended-neutral-100;
+				border-radius: 4px;
```
Comment (creador-dev) [inline packages/assets/css/src/scss/_components/_selector/_elements/_selector-delete.scss:26]: @abder same for this

File: packages/assets/css/src/scss/_components/_checkbox/_elements/_checkbox-box.scss (hunk 1 above)
Comment (abder) [reply packages/assets/css/src/scss/_components/_checkbox/_elements/_checkbox-box.scss:19]: @emgk there's a style set by common styles in: .storybook/assets/css/wordpress/common.css which I wanted to override

File: packages/assets/css/src/scss/_components/_selector/_elements/_selector-delete.scss (hunk 5 above)
Comment (abder) [reply packages/assets/css/src/scss/_components/_selector/_elements/_selector-delete.scss]: Thanks, fixed them

**Review by emgk — APPROVED** [review]

**Pull request merged**